linode/lke/schema_resource.go

```diff
@@ -143,8 +144,48 @@ var resourceSchema = map[string]*schema.Schema{
 					Optional:    true,
 					Computed:    true,
 				},
+				"acl": {
```

Comment:
I noticed dropping the `acl` block after applying a configuration with a valid ACL doesn't trigger any updates. I think this behavior is probably fine/expected but I just wanted to bring attention to it just in case 🙂